arangod/Replication/Syncer.cpp

```diff
@@ -403,7 +403,7 @@ SyncerId newSyncerId() {
     return SyncerId{TRI_NewServerSpecificTick()};
   }
 
-  return SyncerId{ServerIdFeature::getId()};
+  return SyncerId{0};
```

Comment:
mmh, is that really intentional?
If so, IMHO it mandates a comment.

Reply:
Yes, it's intentional. I've added a comment.